### ISSUE TITLE
active?

### ISSUE THREAD

**Issue by ghost** [alexnask/iguanaTLS#31]
you still working on this?

**Comment by elerch** [comment]
Not the author so I can't answer specifically, but zig 0.11.0 now has TLS support: https://github.com/ziglang/zig/blob/master/lib/std/crypto/tls.zig

Note that the crypto has TLS 1.2 references, but TLS 1.3 is all that's supported at the moment: https://github.com/ziglang/zig/issues/14172

**Comment by 1268** [comment]
yeah that's kind of the whole point of this issue. the current Zig TLS support is basically useless, as at least currently TLS 1.2 still sees much wider use, and servers like that will request TLS 1.3 clients.

**Comment by nixigaj** [comment]
I am sorry to inform you that Alexandros Naskos has sadly passed away.

![Screenshot from 2023-09-28 20-54-05](https://github.com/alexnask/iguanaTLS/assets/46706140/dff64500-c0a2-4de4-aa39-887f3424128d)

**Comment by ghost** [comment]
oh wow that is a huge blow. the TLS stack is important to the entire Zig ecosystem, and this is one of if not the only current implementation. I would be willing to pick up the work, but I don't know if he gave anyone else access to his account. sad stuff.

**Comment by elerch** [comment]
+1000, this is a huge loss
